config/helpers.ts

```diff
@@ -201,6 +203,7 @@ export const getFileMetadata = (file: string, root: string) => {
 
     return Observable.of<SnippetFileInput>({
         relativePath: relativePath,
+        type: /private-samples/.test(file) ? 'private' : 'public',
```

Comment:
Strengthened the regex and changed from testing `file` to testing `root`, which only contains the path-names.